charts/kyverno/templates/policies/default/disallow-adding-capabilities.yaml

```diff
@@ -1,14 +1,14 @@
-{{ $name := "disallow-add-capabilities" -}}
+{{- $name := "disallow-add-capabilities" }}
 {{- if eq (include "kyverno.podSecurityDefault" (merge (dict "name" $name) .)) "true" }}
 apiVersion: kyverno.io/v1
 kind: ClusterPolicy
 metadata:
-  name: {{ $name }}
+  name: disallow-add-capabilities
```

Comment:
Do we want to hard code the name here? I didn't see this for other policies.

Reply:
Good catch, that was something I accidentally committed while trying to debug the original issue. I've reverted back to using `$name` variable.